File: python/zlozonosc.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-


def wypisz_nieparzyste(y):
    for i in range(1, y, 2):
        print(i)


def main(args):
    y = int(input(" Podaj wartosc liczby n: "))

    wypisz_nieparzyste(y)

    return 0


if __name__ == '__main__':
    import sys
sys.exit(main(sys.argv))

File: c++/zlozonosc.cpp


#include <iostream>

using namespace std;

void wypisz_nieparzyste(int x)
{
    for(int i=1;i<=x;i+=2)
    {
        cout<<i<<endl;
    }
}

int main(int argc, char **argv)
{
    int x=0;
    cout<<"Podaj wartosc liczby n:";
    cin>>x;
    
    wypisz_nieparzyste(x);
    
    return 0;
}
